### ISSUE TITLE
Retreive function

### ISSUE THREAD

**Issue by dfalster** [ropenscilabs/ozrepro#36]
We want function to reinstate the status of project at previous commit - taking that and placing it on top of current state. This is one way of achieving it, via  a branch and merge.

First go back in time
```
git checkout 7c71ab1
```

Then make some changes. When you try to commit we discovered we're in detahced head
Then 
```
git branch -b reinstate
git add --all
git commit -m "Message" 
git merge master -s ours -m "XXXX"
git checkout master
git merge --ff-only reinstate
git branch -d reinstate
```

**Comment by dfalster** [comment]
Ok, had trouble getting git2r to make the merge happen.

now going for solution [described here](https://stackoverflow.com/questions/1463340/revert-multiple-git-commits), based on `revert --no-commit master~2..master`

**Comment by dfalster** [comment]
Implemented in #46
